src/features/CourtCaseFilters/AppliedFilters.tsx

```diff
@@ -16,6 +16,7 @@ interface Props {
     urgency?: string | null
     locked?: string | null
     caseState?: string | null
+    myCaseState?: string | null
```

Comment:
Just a suggestion, could we name this variable `myCases` instead? The `state` bit feels redundant